diff -up scstadmin-2.2.1/init.d/scst.iscsi-scst-path scstadmin-2.2.1/init.d/scst
--- scstadmin-2.2.1/init.d/scst.iscsi-scst-path	2013-01-06 03:43:28.278469036 -0600
+++ scstadmin-2.2.1/init.d/scst	2014-07-30 01:49:06.721023898 -0500
@@ -159,7 +159,7 @@ parse_scst_conf() {
                 SCST_OPT_MODULES="crc32c-intel $SCST_OPT_MODULES";;
         esac
         SCST_OPT_MODULES="crc32c $SCST_OPT_MODULES"
-        SCST_DAEMONS="/usr/local/sbin/iscsi-scstd $SCST_DAEMONS"
+        SCST_DAEMONS="/usr/sbin/iscsi-scstd $SCST_DAEMONS"
     fi
 }
 
